shoop/locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -0,0 +1,559 @@
+# English translations for PROJECT.
```

Comment:
[Django-REST-Framework directly uses the tx client for uploads.](https://github.com/tomchristie/django-rest-framework/commit/7913947757b0e6bd1b8828db81933c32c498e20a) I think that might be better than storing generated files in the repo.